src/flask_pyoidc/pyoidc_facade.py

```diff
@@ -52,9 +55,7 @@ def __init__(self, provider_configuration, redirect_uri):
         """
         self._provider_configuration = provider_configuration
         self._client = Client(client_authn_method=CLIENT_AUTHN_METHOD)
-        # Token Introspection is implemented in extension sub-package of the
-        # client in pyoidc
-        self._client_extension = ClientExtension(client_authn_method=CLIENT_AUTHN_METHOD)
+        self._token = None
```

Comment:
Is there any way around having to store this here?
`PyoidcFacade` is intended to be stateless, all data to be persisted is handled in layers above this (and finally ends up in Flask session).
This is making me very wary about this changeset - the statefulness of pyoidc is a bit of pretty big downside going this route... 🤔